week1/homework/src/server.js

```diff
@@ -1,17 +1,36 @@
-'use strict';
+"use strict";
 
-const http = require('http');
+const http = require("http");
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
 function createServer(port) {
   let state = 10;
-
+  function okResponse(response) {
```

Comment:
Try not to define functions inside of other functions. `okResponse` here gets defined every time `createServer` is called...